src/Display/Tooltip/Tooltip.tsx

```diff
@@ -52,7 +52,6 @@ export const Tooltip: FC<Props> = ({
       className={classNames('aries-tooltip', classes.container)}
       role="tooltip"
       aria-hidden={isShow ? 'false' : 'true'}
-      aria-label={text}
```

Comment:
Probably don't wanna remove this without a good reason: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

![Screenshot from 2021-09-14 08-27-05](https://user-images.githubusercontent.com/3341028/133174229-2b31f1de-85f1-4c03-8799-1d12069e4052.png)
![Screenshot from 2021-09-14 08-26-43](https://user-images.githubusercontent.com/3341028/133174231-261918a3-5a58-46e8-9a4d-c010fa5d1760.png)

Maybe this was the reason why the Aries team didn't allow `ReactNode` before. The only solutions I can think of are:
* Get the `innerText` of `text`: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/innerText
* Somehow allow passing in a pure string representation of the tooltip through a prop

Reply:
Oh. Now I understand why we are not using `ReactNode`. Let me see what I can do.

Reply:
Yes I think this is the best we can do at the moment.

But we should still make sure that pre-existing `string` `text`s still get used for the label, for backwards-compatibility. So if `textAsString` is not given, it should use `text` for the label (should type-check first though).

Reply:
Alright, I just made a change.

- With string, it will be (textAsString is optional)
```javascript
<Tooltip
    position="top"
    text="Lorem Ipsum"
  >
    <Tag>
      Glints Aries
    </Tag>
</Tooltip>
```

- With html, it will be (textAsString is required)
```javascript
<Tooltip
    position="top"
    text={<b>Hello World</b>}
    textAsString="Hello World"
  >
    <Tag>
      Glints Aries
    </Tag>
</Tooltip>
```

Reply:
Hi, nice solution! Just one more thing: There's a lot of duplication between the two interfaces now. To fix that, I'd suggest you create an interface `BaseProps` which `HtmlTooltip` and `TextTooltip` extend!